### ISSUE TITLE
Cex Extends Api Scalability Suggestions

### ISSUE THREAD

**Issue by rzyaly** [Nouzan/exc#32]
In addition to the general method, can you refer to the project  Position to implement a comm utils lib  for cex extends api request . 
With it , we can use uniform parameters,such as uri and payload to implement method quickly.For example, sometime we need to transfer funds between sub-accounts.

Although I may be a little ambiguous, this is a good idea. It can help users quickly connect to non-core interfaces

**Comment by Nouzan** [comment]
Yes, that is a good idea. I will add such utils to `exc_core` in the future version to help people add support to the exchange APIs they want, and even new exchanges.
